tuftool/tests/download_command.rs

```diff
@@ -95,19 +107,18 @@ fn download_command(metadata_base_url: Url, targets_base_url: Url) {
 // Ensure that the download command works with http url, and that we truncate files when downloading into a non-
 // empty directory (i.e. that issue #173 is fixed).
 fn download_command_truncates_http() {
-    // let repo_dir = utl::test_data().join("tuf-reference-impl");
-    let _role_1 = create_successful_get_mock("metadata/role1.json");
-    let _role_2 = create_successful_get_mock("metadata/role2.json");
-    let _snapshot = create_successful_get_mock("metadata/snapshot.json");
-    let _targets = create_successful_get_mock("metadata/targets.json");
-    let _timestamp = create_successful_get_mock("metadata/timestamp.json");
-    let _file1 = create_successful_get_mock("targets/file1.txt");
-    let _file2 = create_successful_get_mock("targets/file2.txt");
-    let _file3 = create_successful_get_mock("targets/file3.txt");
-    let base_url = Url::from_str(mockito::server_url().as_str()).unwrap();
-    base_url.join("metadata").unwrap().to_string();
-    let metadata_base_url = base_url.join("metadata").unwrap();
-    let targets_base_url = base_url.join("targets").unwrap();
+    let server = Server::run();
+    server.expect(create_successful_get("metadata/role1.json"));
+    server.expect(create_successful_get("metadata/role2.json"));
+    server.expect(create_successful_get("metadata/snapshot.json"));
+    server.expect(create_successful_get("metadata/targets.json"));
+    server.expect(create_successful_get("metadata/timestamp.json"));
+    server.expect(create_successful_get("targets/file1.txt"));
+    server.expect(create_successful_get("targets/file2.txt"));
+    // TODO - server.expect(create_successful_get("targets/file3.txt")); when delegate support lands.
```

Comment:
where is this TODO coming form ? Do we need it ?

Reply:
It was there before these changes. It can be deleted. I think it might be true that this test has to be changed if we support delegates, but I'm not sure. Or maybe delegates are already supported? I forget. Anyway it's probably not a helpful TODO statement.